bagit/reader.go

```diff
@@ -5,10 +5,9 @@ import (
 	"bufio"
 	"encoding/hex"
 	"errors"
+	"github.com/ndlib/bendo/util"
```

Comment:
I was keeping the local packages separate from the standard library packages. it is a local style thing. I'll add it to the readme.